packages/openapi-generator/src/file-serializer/operation.spec.ts

```diff
@@ -102,23 +104,24 @@ describe('serializeOperation', () => {
         additionalProperties: { type: 'any' },
         properties: []
       },
-      pathPattern: 'test/{id}'
+      pathPattern: '/test/{id}'
     };
 
-    expect(serializeOperation(operation)).toMatchInlineSnapshot(`
+    expect(serializeOperation(operation, apiName)).toMatchInlineSnapshot(`
       "/**
-       * Create a request builder for execution of delete requests to the 'test/{id}' endpoint.
+       * Create a request builder for execution of delete requests to the '/test/{id}' endpoint.
        * @param id - Path parameter.
        * @param headerParameters - Object containing the following keys: resource-group.
        * @returns The request builder, use the \`execute()\` method to trigger the request.
        */
       deleteFn: (id: string, headerParameters?: {'resource-group'?: string}) => new OpenApiRequestBuilder<Record<string, any>>(
         'delete',
-        "test/{id}",
+        "/test/{id}",
```

Comment:
Drive by change, as all path parameters [should begin](https://swagger.io/specification/#paths-objec) with a `/`  according to Swagger v3.

Reply:
I agree. In general, path should contain at least one '/' at the beginning, even `http://example.com` with no trailing slash has path `/` instead of ''.

I suppose you adapted all occurrences.

Reply:
Yes, I adapted it everywhere in the this file's tests.